src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -208,8 +225,8 @@ export default class FiatConnectQuote extends NormalizedQuote {
     return this.quote.quote.cryptoType
   }
 
-  getCryptoType(): Currency {
-    return resolveCurrency(this.quote.quote.cryptoType)!
+  getCryptoType(): CiCoCurrency {
+    return this.quote.quote.cryptoType as unknown as CiCoCurrency
```

Comment:
is this a safe typecast? would it be safer to just use a map to convert?

Reply:
yes, good idea